postgrid/__init__.py

```diff
@@ -473,6 +554,93 @@ def construct_event(cls, payload, secret):
             raise ValueError()
 
 
+class LookupInfo(BaseResource):
+    @classmethod
+    def lookup_info(cls):
+        return _av_request('', method='GET')
+
+
+class Verification(BaseResource):
+    @classmethod
+    def verify(cls, address=None):
+        return _av_request('verifications', method='POST', json={'address': address})
+
+    @classmethod
+    def batch_verify(
+        cls, raw_body=None, include_details=True, proper_case=True, geocode=True
+    ):
+        return _av_request(
+            endpoint='verifications/batch',
+            method='POST',
+            params={
+                'include_details': include_details,
+                'proper_case': proper_case,
+                'geocode': geocode,
+            },
+            json=raw_body,
+        )
+
+
+class Autocomplete(BaseResource):
+    @classmethod
+    def previews(
+        cls,
+        partial_street=None,
+        city_filter=None,
+        state_filter=None,
+        pc_filter=None,
+        country_filter=None,
+        prov_instead_of_pc=None,
+        proper_case=None
+    ):
+        return _av_request('completions', method='GET', params=locals())
+
+    @classmethod
+    def address(
+        cls,
+        partial_street=None,
+        city_filter=None,
+        state_filter=None,
+        pc_filter=None,
+        country_filter=None,
+        index=None,
+    ):
+        return _av_request(
+            endpoint='completions',
+            method='POST',
+            data={
+                'partial_street': partial_street,
+                'city_filter': city_filter,
+                'state_filter': state_filter,
+                'pc_filter': pc_filter,
+                'country_filter': country_filter,
+            },
+            params={'index': index},
```

Comment:
If `index` is `None`, does this pass along `null` for the index?

Reply:
Looks like it doesn't get passed at all